### ISSUE TITLE
tell users about incoming number

### ISSUE THREAD

**Issue by kra** [futel/asteriskserver#584]
Where do we even tell users about the incoming number besides the about voicemail context?

Maybe there should be a more general about futel context. Connect with the incoming line at 503 468 1337 or from any Futel phone. For a map of Futel phones, see futel.net.

**Comment by breedx2** [comment]
This is rather insightful/interesting. I had assumed that the incoming line was not loudly advertised for some historical/artful reason.

@kra do you think it's also worth raising an issue on the website repo to put the incoming line somewhere more obvious as well?

**Comment by kra** [comment]
@breedx2 yes thanks, I added a few related issues https://github.com/kra/futel.net/issues

I've never really advertised it because it's maybe not the best face of the project and because it was always more fun to not explain anything. And less importantly because it's another attack surface. But it has what's supposed to be one of the primary services. And someone emailed me saying they forgot it.

We can always make another secret number :)
